packages/epo-widget-lib/.storybook/main.js

```diff
@@ -8,6 +8,7 @@ const config = {
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
     "@storybook/addon-jest",
+    "@storybook/addon-a11y",
```

Comment:
added the storybook a11y addon, no other significant setup right now just nice to have